### PR TITLE
20 fix: add lint stage config

### PR DESCRIPTION
**Pull request by PengchengYuGithub**
Add lin-staged config file named .lintstagedrc.js. The calling order is husky pre-commit => lint-staged => eslint & prettier.
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "coverage": "jest --coverage",
     "prettier": "prettier --check .",
     "prettier:fix": "prettier --write .",
-    "pre-commit": "npm run prettier:fix && npm run lint && npm run test"
+    "lint-staged": "npx lint-staged",
+    "pre-commit": "npm run lint-staged && npm run test"
```
Comment (tatoMa) [inline package.json:16]: Where is prettier? Or you put it somewhere else?

Comment (PengchengYuGithub) [reply package.json:16]: Next suggests running next lint in a defined buildEslintCommand in .lintstagedrc.js; I also put prettier command in that document.

**Comment by Double-Twelve** [comment]
should `.lintstagedrc.json` be removed?

**Comment by PengchengYuGithub** [comment]
.lintstagedrc.json removed @Double-Twelve

**Review by tatoMa — APPROVED** [review]
LGTM

**Pull request merged**